utils.py

```diff
@@ -0,0 +1,28 @@
+"""
+This file contains various common utilities that facilitate gear excecution
+"""
+import logging
+import sys
+
+def escape_shell_chars(path):
```

Comment:
Could replace with:
```
import re
path = re.sub('[^0-9a-zA-Z]+', '_', path)
```
for quicker execution and avoiding long repeats of underscores.

Reply:
Thanks, Kaleb,
With some modifications, that will be much faster and take into consideration a full path (e.g. /path/to/my/file.nii.gz) for shell-safe characters that excluding all non-alphanumeric would miss.